compositional_skills/technology/qna.yaml

```diff
@@ -0,0 +1,37 @@
+created_by: lzrvc-ibm
```

Comment:
While LLMs can do simple reasoning and counting out of the box, solving complex tasks like linear equations with multiple variables, or matrix inversion and multiplication belongs to ["core skill"](https://github.com/instructlab/community/blob/main/docs/SKILLS_GUIDE.md#core-skills), which is not open to contribution. 

However, if you want to limit this example to very simple two-variable linear equations, and can be solved by with step-by-step reasoning, and tricks like substitution and elimination, then it could be eligible. So feel free to try it again as a new PR if you are interested!